### ISSUE TITLE
Null conventions rework

### ISSUE THREAD

**Issue by TheDaniMan** [hanabi/hanabi.github.io#975]
Edit: I changed null to not always be a hard variant. Please see my pull request at https://github.com/hanabi/hanabi.github.io/blob/f223ac8c44ba065df1de546fd8515286a0553d6e/docs/variant-specific/null.mdx
for the current proposal (suggestions for changes are still welcome).

_______________________________________________
A few months ago I floated the idea to completely rework the hyphenated null conventions, but ended up waiting with it because there were some other possibly upcoming conventions that could have conflicted with my ideas. In the time since, I've played multiple null games, and each time we played without the hyphenated null conventions (not even by my request - other players don't seem to like them either). I would therefore like to codify how these games have been played, and add some conventions to improve quality of life a bit. The following are my suggested conventions:

Null positional clues would remain unchanged, but would see much rarer use. Aside from the double save exception, they may only be given by Alice to Bob. The following are null positional clues:
1) Clues to trash cards - if a clue cannot be any of trash chop move, double ignition, playful ignition, unknown trash/dupe discharge, or trash push, then it must be a null positional clue. This null positional is to be interpreted as in the current conventions. The clue recipient should first discard the focus of the clue, and then if not fixed discard all remaining cards touched by the clue.
2) Rank clues to the highest available rank - if such a clue is given which is not a 5-save, a 5s chop move, rank choice ejection, 4 double bluff (with Cathy's finesse position immediately playable), or a normal play or save clue that could not have been given with color without causing bad touch, then this is a null positional to Bob's chop card. In the special case that this is done with a 5 clue that looks like a 5s chop move, the null positional must be demonstrated immediately, whether by blindplaying or by chop moving and discarding.

Upon receiving a null positional clue, if the clue is of type (1), it is always a play clue. Bob should immediately play the indicated slot. If the played card is not a null card and could've been otherwise clued, the move is considered unnecessary, and the clue receiver should trash push / cm as applicable. If the clue is of type (2) and Bob is loaded, Bob should play the rightmost card in his hand that can be a null card. Otherwise, Bob should chop move. Note that case (2) is typically done with the number 4 clue in variants such as Brown-fives, or if there are no apparent 5s to clue.

In order to accommodate for these, the following conventions are turned off: Rank choice ejections with #2, trash push ejections, bad chop move ejections, trash ejections, replay ejections, poke ejections, trash push discharges, unknown trash charm, 5 pulls, 5 numbers discharges.

Scream blind plays for two chop moves remain, but the double-save is offered as an alternative:
Suppose Cathy has a critical null card on her chop, and additional critical cards preceding it. If it is not possible to save them with one clue (e.g. a 5s chop move or a trash chop move), but it is possible to save them with two (e.g. a null chop move followed by a save clue), then Alice may initiate a double-save by giving a rank clue to the highest available rank. In this specific case, instead of chop moving, Bob should then save the remaining critical cards on Cathy, and Cathy should always assume that she has been double-saved (as opposed to Alice giving Bob a save clue and Bob giving Cathy a play clue - in such a case, Alice should discard).

Notably, if the critical cards could have been saved with one clue, Bob should instead assume that the chop move is on him. On the other hand, if they cannot be saved even with two clues, then Bob should assume that the null chop move applies to him, and that he is intended to immediately scream blind play to prevent Cathy from discarding. Players should not attempt to null chop move Bob and Cathy consecutively unless both have a very good way of knowing what is happening.

To avoid confusion with double-saves, saving the null 2 is optional, and in most cases players should not write elimination notes for the null 2 when it is discarded. Similarly, the first null 1 is permitted to be discarded, as long as it was in the starting hand. Non-critical null 3 and 4 may be saved, but never by a null positional clue.

**Comment by argothiel** [comment]
ad. 2 - this also turns off double bluffs and double finesses with 4 rank, if Bob doesn't see any available 5s?

**Comment by argothiel** [comment]
ad. 2 - the highest available rank can be e.g. 3, if there are no 4s nor 5s available?

**Comment by argothiel** [comment]
Maybe it'd be better to explicitly separate null 5-tech and null 4-tech?

**Comment by Zamiell** [comment]
i am honestly surprised that no-one wanted to make a pull request for the dawn stuff, since it seemed to be particularly effective for null.

as for this proposition, it seems fine to me, i dont really play a lot of null myself

**Comment by TheDaniMan** [comment]
I think 4DB should take precedence when Bob sees playable on Cathy's finesse (I will add that in), but certainly 4 charms with rank are off. It's possible for 3 to be the highest ranks, but much like first turn 5 stalls in hard variants we're gonna pretend this doesn't happen.

**Comment by newduke** [comment]
+1 

I have been playing null without h-group null, but have occasionally fallen back to positional clues "when it makes sense." What you have laid out codifies the logic in a clear, consistent fashion (afaict with no play testing).

How does Roger know that the rank clue was a CM on bob (in the ambiguous case of u12 especially)? Why doesn't Roger just play the clued card in these cases?

**Comment by argothiel** [comment]
Can you also add that
_Double Finesses_, _Triple Finesses_ and _Triple Bluffs_ with 4 rank clue are also off when the very next player has to blind-play two or more cards and there is no 5 available?

Also, there are missing Double/Triple/Quadruple Finesses and Double/Triple Bluffs with 5 rank clue in the list.

**Comment by iamwhoiamhahaha** [comment]
looks pretty similar to the current framework

**Comment by pianoblook** [comment]
I can't help but reshare our [non-positional Null rework](https://docs.google.com/document/d/1idE_9n0y34B8-4BGJCtrAqF47pLi7s9sNOQ3UpDUWv8/edit#heading=h.ds5ky0b6ao6e), which I'm still a firm believer of.

Benefits:
- 5 tech and trash tech are fully on
- No HVR (meaning 5 stalls work --> 5 reclues work)
- No confusion around positional vs not

**Comment by Zamiell** [comment]
To be more concrete, it seems like Piano is proposing that the following four moves are added to the document (for null):

- *Null 5 Chop Move* - Re-cluing a 5.
- *Null Trash Chop Move* - Re-cluing known trash.
- *Loaded 5 Null Play Clue* - Play oldest if *Loaded*.
- *Null Ignition Kicks* - *Replay/Poke Ejection* --> *Kick*
- (Dawn is not added.)

And then positional clues are completely turned off.

So I guess this doesn't overlap with anything that Dani is proposing, correct me if I am wrong.

**Comment by Zamiell** [comment]
Dani/argothiel/Duke/Jeff/whoever, what are your thoughts on that?

**Comment by TheDaniMan** [comment]
> looks pretty similar to the current framework

The point of this is that it is completely different from the current hyphenated null conventions.

**Comment by TheDaniMan** [comment]
> +1
> 
> I have been playing null without h-group null, but have occasionally fallen back to positional clues "when it makes sense." What you have laid out codifies the logic in a clear, consistent fashion (afaict with no play testing).
> 
> How does Roger know that the rank clue was a CM on bob (in the ambiguous case of u12 especially)? Why doesn't Roger just play the clued card in these cases?

I had a few iterations of ideas about this, including:
1) null chop moves must be demonstrated immediately, i.e. Bob must discard right after being chop moved.
2) play clues with highest number are just always off, and those should be clued with colors instead.
The one I'm currently hinting at is
3) Alice should refrain from giving direct play clues or self-finesses to highest ranks when Bob is not loaded and has a chopmoveable null card.

**Comment by TheDaniMan** [comment]
> To be more concrete, it seems like Piano is proposing that the following four moves are added to the document (for null):
> 

I like the clarity in pianoblook's proposal, but I am concerned about the possibility of no clued 5s/known trash, especially at the start of the game. We rarely hold for a long time or even have trash in the early game, and aside from the possibility of having no 5s, recluing a 5 can be very difficult if the 5 didn't otherwise get clued, and might require not just 5 stalling but orchestrating a 5 stall by the right player.
I am also really against not knowing who the chop move pertains to and having to make the two players with null cards on chop chop move each other.
So my counteroffer is to change the first point and to add the Bob condition and doublesave:

> * _Null 5 Chop Move_ -  Rank clue to any 5 (5 save and 5cm take precedence), or to 4 if 5s not visible. All other 5 tech is still off, _including 5 stalls_, unless Bob is loaded. CM applies only to Bob.
> * _Null Trash Chop Move_ - Re-cluing known trash. CM applies only to Bob.
> * _Loaded 5 Null Play Clue_ - If it is the first time the 5 is clued it has whichever normal interpretation (5 stall, 5 pull, 5nd etc). Reclue means play oldest.
> * _Null Ignition Kicks_ - _Replay/Poke Ejection_ --> _Kick_

In addition, doublesaves work the same way as in my original proposal - this is somewhat necessary to add in order to compensate for Bob-only chop moves.

> And then positional clues are completely turned off.

**Comment by TheDaniMan** [comment]
> Regarding piano's conventions: I like the fact that it's already well documented and that all 5 Tech (including 5 Pulls) is still a thing. I also prefer Kicks over positional in case of Poke Ejections. I dislike the fact that the CM is not always on Bob, the fact that you have to waste a clue to do a 5 Null Tech, and that you cannot do it with 4 if there are no 5s visible. But generally, I like it. Note that if we're going to adopt KiPiPi 5-Null Tech without Dawn conventions, we'd have to somehow integrate it with positional clues to make the right card play.
> 
> Regarding DaniMan's conventions: I think this proposal suffers from being a compromise between clarity and usability. I feel like the 5 Null Tech is overly complicated here. I think the 5 Null Tech should take more priority, because it's usually the null suit which will require the most attention. Maybe just make any 5 which is not a 5-save or stalling situation - a chop move on Bob or a kick if he's loaded? Seems simple enough to avoid misunderstanding. And it would still be a huge improvement over existing H-Group convention. This is regarding his proposed point 2 (regarding 4/5-Tech), the point 1 is really nice and clear.

I think you have a good point that my proposal is a bit too complicated to be reasonable for what is, after all, a base variant. I believe I've taken most of your suggestions, but in my opinion:
1) 5 stalls need to be illegal if Bob may be about to discard a null card, since otherwise there's often no way to save them, and
2) we can keep 5 tech, keep 5 null tech and avoid overly complicating by eating into the loaded 5 null play clue; replay/poke kicks seem plenty good enough on their own.

**Comment by T-Hoodie** [comment]
Here's how I've been playing Null-suits lately:

Null 5 Chop Move - If Bob is not loaded, a rank clue to any 5 (aside from 5 save or 5cm), ~~or to a 4 if no 5s are visible,~~ indicate a chop move on Bob. All other 5 tech is still off, including 5 stalls. Only Bob may be chop moved in this way, and the clue recipient should assume good touch but should not assume the clued card is playable.
Null Trash Chop Move - If Bob is not loaded, recluing known trash indicates for Bob to chop move,** if there is no 5 available**.
~~Loaded 5 Tech Clue - If Bob is loaded and a 5 is clued for the first time, this is to be interpreted with normal hyphenated conventions (5 pull, 5 stall, 5nd etc.).~~
Loaded 5 Null Play Clue - If Bob is loaded and a 5 ~~(or 4 if no 5s are visible)~~ are reclued, bob should play his rightmost card that could be a null card.
~~Null Ignition Kicks - Replay ejections and Poke ejections instead indicate for Bob to play his chop (unless Null Trash Chop Move applies).~~
**Doublesave as in Danis proposal: Possible at any time, be wary of context.**
**Positional clues with rank/color: Possible in the endgame, be wary of context.**
**No elim notes for u1 discard**
**If no other saves are available, make Bob bomb a card, and Bob cms all cards to the right of the bombed card.**

**Comment by TheDaniMan** [comment]
> I like what T-Hoodie described - for H-group it's nice to keep it simple and see how it works.
> 
> If you are saying that 5 stalls are off, what do you mean? You cannot clue 5 off-chop in a stalling situation e.g. if you have a locked hand or at 8 clues? Do you have to use hard burn instead?

i.e. if you clue a 5 in this way, Bob should chop move.
Imagine that Bob has u5 on his chop. The rest of the team cannot save it (per the Bob cm convention), so the player before Alice discards. A blue 4 is discarded, putting Alice in DDA. If 5 stalls are on the team will now lose the game.

**Comment by pianoblook** [comment]
> To be more concrete, it seems like Piano is proposing that the following four moves are added to the document (for null):
> [...]

I doubt you could profitably get away with turning off positionals without using Dawn, sadly.
Perhaps you could just agree to have them on during the first few rounds or something, lol, but that sounds like just a worse Dawn.

All I know for sure is that the current Null conventions are a nightmare for clear & consistent play, and its usefulness has long since been outclassed by all the new tech developed over the past 2 years.

**Comment by TheDaniMan** [comment]
> Here's how I've been playing Null-suits lately:
> 
> Null 5 Chop Move - If Bob is not loaded, a rank clue to any 5 (aside from 5 save or 5cm), ~or to a 4 if no 5s are visible,~ indicate a chop move on Bob. All other 5 tech is still off, including 5 stalls. Only Bob may be chop moved in this way, and the clue recipient should assume good touch but should not assume the clued card is playable. Null Trash Chop Move - If Bob is not loaded, recluing known trash indicates for Bob to chop move, _if there is no 5 available_. ~Loaded 5 Tech Clue - If Bob is loaded and a 5 is clued for the first time, this is to be interpreted with normal hyphenated conventions (5 pull, 5 stall, 5nd etc.).~ Loaded 5 Null Play Clue - If Bob is loaded and a 5 ~(or 4 if no 5s are visible)~ are reclued, bob should play his rightmost card that could be a null card. ~Null Ignition Kicks - Replay ejections and Poke ejections instead indicate for Bob to play his chop (unless Null Trash Chop Move applies).~ _Doublesave as in Danis proposal: Possible at any time, be wary of context._ _Positional clues with rank/color: Possible in the endgame, be wary of context._ _No elim notes for u1 discard_ _If no other saves are available, make Bob bomb a card, and Bob cms all cards to the right of the bombed card._
> 
> Edit: formatting

Since the differences are relatively few, let me discuss why I think they are significant.
1) "make Bob bomb a card" may lead to Bob mistakenly guessing that he was meant to play another card, or may make Bob chop move when he accidentally plays the wrong card and a chop move was not intended. In other words, it is the kind of convention that has the potential to snowball a mistake into a game loss, and I'd like to avoid those (hence I also removed scream blindplay - just a couple of games ago I had a player play finesse position because that was his chop, and the next player ignored it).
2) "Doublesave as in Danis proposal: Possible at any time, be wary of context." - in practice it will probably turn out this way, but I think conventions should be written a little more strictly to give precise instructions for how to handle trickier situations.
3) no 4 tech - this is mostly for 3 player, where it is now not uncommon to have no ability to save null cards. There is basically a choice between being able to give play/save clues with number 4 and being able to save null cards, and I think the latter is more important.
4) loaded 5 tech off - the rest of the comments suggested high affinity for 5 tech, this seemed like a pretty easy compromise, but I agree that the value is not that high and the conventions are made simpler by taking this out.
5) kicks: a compromise, but I think a sensible one given that we're more likely to want chop played than second finesse position. the latter could be easily clued the turn before or after, whereas chop can be hard to get if it is a null card.
6) lategame positional clues: I don't mind these, but they greatly increase complexity and it is somewhat hard to define when they are turned on.
Anyway you did write that that is how you play, but what do you think would be good for conventions?

**Comment by TheDaniMan** [comment]
> > To be more concrete, it seems like Piano is proposing that the following four moves are added to the document (for null):
> > [...]
> 
> I doubt you could profitably get away with turning off positionals without using Dawn, sadly. Perhaps you could just agree to have them on during the first few rounds or something, lol, but that sounds like just a worse Dawn.

It seems like a massive waste to define positionals for the first two rounds, and I agree that this is just a way worse Dawn. I'm open to incorporating Dawn, but, wasn't it supposed to be added for all variants not too long ago?

> 
> All I know for sure is that the current Null conventions are a nightmare for clear & consistent play, and its usefulness has long since been outclassed by all the new tech developed over the past 2 years.

I couldn't agree more.

**Comment by argothiel** [comment]
> 1. "make Bob bomb a card" may lead to Bob mistakenly guessing that he was meant to play another card, or may make Bob chop move when he accidentally plays the wrong card and a chop move was not intended. In other words, it is the kind of convention that has the potential to snowball a mistake into a game loss, and I'd like to avoid those (hence I also removed scream blindplay - just a couple of games ago I had a player play finesse position because that was his chop, and the next player ignored it).

Misplay Chop Move is already a thing. This is nothing new.

> 5. kicks: a compromise, but I think a sensible one given that we're more likely to want chop played than second finesse position. the latter could be easily clued the turn before or after, whereas chop can be hard to get if it is a null card.

By the way, it's weird to me that kicks work on chop instead of the rightmost possible card (including chop moved cards).

**Comment by Zamiell** [comment]
> wasn't it supposed to be added for all variants not too long ago

i asked for piano (or anyone, really) to submit a PR on Dec 31, 2021

noone had done so by March 28th (roughly 3 months later), which indicated to me that no-one wanted it added, so i closed the issue, as it was blocking other convention discussions

**Comment by TheDaniMan** [comment]
> Misplay Chop Move is already a thing. This is nothing new.
> 

The way it is often played is with any misplay, e.g. cluing a bomb 5ce to misplay chop move the next player, which in that particular case is usually okay but can otherwise be very confusing. If we are talking about misplay chop move then of course we don't need to include it a second time in variant specific (although I can't say I'm a fan of a 0-for-2 convention under any circumstances).

> By the way, it's weird to me that kicks work on chop instead of the rightmost possible card (including chop moved cards).

I don't have that much of an opinion on it, I just tried to copy the linked document. One good reason that comes to mind is that you might cm u5 and then find a playable null card gets to chop, and then you want a way to get cm'd cards and a different way to get chop.

**Comment by T-Hoodie** [comment]
>    Anyway you did write that that is how you play, but what do you think would be good for conventions?
I wanted to give some context, how nulls are being treated currently in my games. For Conventions I would say:

> 1. "make Bob bomb a card" may lead to Bob mistakenly guessing that he was meant to play another card, or may make Bob chop move when he accidentally plays the wrong card and a chop move was not intended. In other words, it is the kind of convention that has the potential to snowball a mistake into a game loss, and I'd like to avoid those (hence I also removed scream blindplay - just a couple of games ago I had a player play finesse position because that was his chop, and the next player ignored it).
Bomb chop moves don't happen very often, because they usually look like a mistake - Using bombs this way is both unreliable and uncommon in normal games. I was trying to emphasize that in Null Variants a bomb chop move is not as uncommon, and you shouldnt jump to the conclusion of a known mistakes as quickly as normally.
That being said, they are not common in Null Variants either, mostly only needed if you have u5 and no other 5 anywhere in hand.
> 2. "Doublesave as in Danis proposal: Possible at any time, be wary of context." - in practice it will probably turn out this way, but I think conventions should be written a little more strictly to give precise instructions for how to handle trickier situations.
Your formulation seems good, was just saying that this also can happen in my games.
> 3. no 4 tech - this is mostly for 3 player, where it is now not uncommon to have no ability to save null cards. There is basically a choice between being able to give play/save clues with number 4 and being able to save null cards, and I think the latter is more important.
Not a fan of 4 tech, bomb cms are enough imo.
> 4. loaded 5 tech off - the rest of the comments suggested high affinity for 5 tech, this seemed like a pretty easy compromise, but I agree that the value is not that high and the conventions are made simpler by taking this out.
Nothing to add.
> 5. kicks: a compromise, but I think a sensible one given that we're more likely to want chop played than second finesse position. the latter could be easily clued the turn before or after, whereas chop can be hard to get if it is a null card.
Nothing to add.
> 6. lategame positional clues: I don't mind these, but they greatly increase complexity and it is somewhat hard to define when they are turned on.
A strict convention/ruleset for this probably gets convoluted quickly because they are so dependant on context. I would be ok with not having it written down in the doc, I'm convinced players will get the hang of them just by playing nulls.

**Comment by TheDaniMan** [comment]
> > Anyway you did write that that is how you play, but what do you think would be good for conventions?
> 
> I wanted to give some context, how nulls are being treated currently in my games. For Conventions I would say:
> 
> > blah
> 
> blah
> 
> 
> blahblah

Correct me if I'm wrong, but I read this as "I like all of your proposal except for 4 tech, which is too confusing".
That seems to be a common sentiment, so I'm thinking to either put it as a footnote or just remove it altogether. I personally think it's nice to have a fallback that is almost never used to be prepared for edge cases, but that doesn't seem to be what everyone else here thinks about 4 tech.

**Comment by T-Hoodie** [comment]
Yes, thats the essence of it. Good summary XD

**Comment by argothiel** [comment]
Yes, I would say drop 4-tech, except for the variants where 5 clues are not available (Brown-Fives). You might want to describe it separately on the variant's page.

**Comment by TheDaniMan** [comment]
We usually don't have conventions for "_ & _" variants, because there are way too many cases and if it's just for the sake of those ones then the players can figure it out before starting; perhaps it's worth a one line entry in special-fives about null-fives.

**Comment by pianoblook** [comment]
> i am honestly surprised that no-one wanted to make a pull request for the dawn stuff, since it seemed to be particularly effective for null, and would presumably render positional clues unnecessary.

As I mentioned in #589,  I didn't feel comfortable officially adding it without more community support. That said, I'm *also* honestly surprised there wasn't more interest, haha - I think @Dr-Kakashi's strongly voiced objections maybe spooked some people, I dunno. (Though @sjdrodge and I tried our best to counter them)

Despite how good it's proven to be, it hasn't felt like there has been a sufficient, widespread, interest in adopting it. TBF it changes the start of the game quite a bit, so I totally get it.

To be very clear, I firmly believe it's much stronger than non-Dawn, in any variant (but particularly in Brown/Omni/Null). Funnily enough, we've already revised Dawn (see the [New Dawn](https://docs.google.com/document/d/1i9sDpQtch6VRdBa2lTGPThWEbvtXFP6pkrdQd_z90gY/edit?usp=sharing) doc just published this week)

**Comment by pianoblook** [comment]
If you haven' tried Dawn Null though, I urge you to give a whirl; it's honest-to-goodness my favorite variant type now. I really wish I could play it with a broader group D':

**Comment by argothiel** [comment]
@pianoblook Maybe you should do open playtesting more often to encourage people, like the ref sieve gang is doing? :)

**Comment by Zamiell** [comment]
have we arrived on a consensus of which moves specifically to turn on and turn off?

**Comment by TheDaniMan** [comment]
Based on some playtesting, I'd like to take out the "Null Trash Chop Move", since it's pretty much never used but has on a couple of occasions ruined some attempted play clues later in the game; trash is usually held only at the later stages, at which point normal trash chop moves can be used when a play is not intended, and conversely it gets harder to clue non-trash cards to get a play if desired.
I also didn't find anyone else who liked 4-tech when not playing null-fives.

**Comment by Romain672** [comment]
Just a random though: How about putting null conventions in two part: 'easy' and 'hard'?

**Comment by TheDaniMan** [comment]
I'm not super keen on this, since people playing null are probably already playing at or very near max level, whereas the only variant specific conventions with levels that I know of (pink) are for a variant that's not that hard but has a wide range of clues that can be given to its suit.

**Comment by Zamiell** [comment]
@pianoblook take a look at dani's PR here:
<https://github.com/hanabi/hanabi.github.io/pull/1001/files>

(also @argothiel @newduke @iamwhoiamhahaha @T-Hoodie )

In short, 

1) null 5 chop move - yes
2) null trash chop move - no - deleted because it was annoying in practice; recluing trash is a kick
3) loaded 5 null play clue - no - this is dani's attempt to keep 5 tech since a couple of people suggested they want to keep 5 tech
4) null ignition kick - yes

thus, I think the PR is good progress towards consensus, please give your thoughts

**Comment by TheDaniMan** [comment]
> 3. loaded 5 null play clue - no - this is dani's attempt to keep 5 tech since a couple of people suggested they want to keep 5 tech
Small note: This still exists, but only with already clued 5s.

**Comment by pianoblook** [comment]
This all looks like a big improvement. I'll toss out a couple thoughts: 

- Personally I'd still prefer to allow/encourage 5 Stalls, and then just have Null 5CMs only be for reclues, but the difference in utility is probably pretty small + I'm happy to support whichever others think is better for the community.

- Favoring Trash Reclues as Kicks > Null TCM is probably good.

- One important detail that I'd add re: Ignition Kicks = does it *promise* a Null card being gotten? As in, can you do a Layered Kick if the first card that blind-plays is non-Null? Personally I'm **in favor** of having it be assumed Layered, but again don't have a strong preference.

**Comment by TheDaniMan** [comment]
Obviously I'm in favor of no 5 stalls, but I'm still curious what other people think of this.

> * One important detail that I'd add re: Ignition Kicks = does it _promise_ a Null card being gotten? As in, can you do a Layered Kick if the first card that blind-plays is non-Null? Personally I'm **in favor** of having it be assumed Layered, but again don't have a strong preference.

This is an interesting point; I remember when I had just joined, positional clues were required to get a null card in this way.
Given that a kick gets a card on chop, which is the easiest slot to get by hyphenated conventions, it seems like it would almost never be needed in order to get a non-null card, so I agree with you on this.

**Comment by argothiel** [comment]
I really like this proposal. Thank you for your work on that!

- I don't have a lot of experience with Null Ignition Kicks and it's maybe unnecessary for beginners playing with null, but I'll get used to it.
- I like no 5-Stalls.
- I'd prefer Loaded 5 Null Play Clue also for unclued 5s instead of Loaded 5 Tech Clue, but I can understand the reasoning.

Regarding the wording, I'd drop the "The" at the beginning of the moves ("Double Save" instead of "The Double Save" etc.).

**Comment by newduke** [comment]
> * I'd prefer Loaded 5 Null Play Clue also for unclued 5s instead of Loaded 5 Tech Clue, but I can understand the reasoning.

Recluing a 5 is still a loaded play.  Do you propose that after low score phase, rank 5 play is off in favor of loaded 5 null play clue?

**Comment by argothiel** [comment]
Yes, until all the nulls are gotten, 5s to play would be reclued only with color.

**Comment by argothiel** [comment]
That's how I understand the current proposal:
"- Note that play clues with a number clue to a previously clued 5 are always off."

**Comment by TheDaniMan** [comment]
All right, I'm starting to think 5 pulls in null are not that desired, and maybe I can fully remove loaded 5 tech clue?

> Recluing a 5 is still a loaded play. Do you propose that after low score phase, rank 5 play is off in favor of loaded 5 null play clue?

I think losing the ability to play clue 5 by rank is really no great loss

**Comment by pianoblook** [comment]
5 Pulls are a lot better with [Precision 5 Tech](https://docs.google.com/document/d/1idE_9n0y34B8-4BGJCtrAqF47pLi7s9sNOQ3UpDUWv8/edit#heading=h.nfoajojqmjv) too. 
It's weird that we never actually proposed that on here...Huh. Precision 5s is super good, especially in suits with a negative constraint.

**Comment by Zamiell** [comment]
please review my edits
